pom.xml

```diff
@@ -82,6 +95,16 @@
                 <groupId>org.codehaus.gmavenplus</groupId>
                 <artifactId>gmavenplus-plugin</artifactId>
                 <version>1.6</version>
+                <configuration>
+                    <testSources>
+                        <testSource>
+                            <directory>src/test/groovy</directory>
```

Comment:
Is it true that test codes is under `src/test/groovy/java`, now?
Shouldn't they be under `src/test/groovy`? 
`java` folder is my question actually

Reply:
I have changed the directoy.